### PR TITLE
feat(Drawer): consumed Penta updates

### LINKED ISSUE #9979: Consume core Penta updates: Drawer
follow up to core https://github.com/patternfly/patternfly/pull/6183

The following was done with core bump PR #10017 

- Removed light color variant form DrawerColorVariant
- disabled out color variant unit test
- removed color variant from drawer integration demo
- disabled drawer background variant integration test
- Removed light variant from Modal with drawer Wizard Demos and In page with drawer wizard Demos
- Removed light variant from drawer panel wizard test
- Removed light variant from WizardWithDrawer example
- disabled failing integration test

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #9979
Codemod: https://github.com/patternfly/pf-codemods/issues/553

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/Drawer/__tests__/Drawer.test.tsx [hunk 1]
```diff
@@ -70,15 +71,14 @@ test(`Drawer expands from bottom`, () => {
   expect(asFragment()).toMatchSnapshot();
 });
 
-// TODO: Update/renable with issue #9979
 xtest(`Drawer has resizable css and color variants`, () => {
```
Comment (wise-king-sullyman) [inline packages/react-core/src/components/Drawer/__tests__/Drawer.test.tsx:74]: Should this test have been reenabled?

**Review by wise-king-sullyman — COMMENTED** [review]
Code all looks good, will have to circle back to this when surge is back up and running to finish the review though.

One small thing I saw that I _think_ needs updating.

**Review by andrew-ronaldson — APPROVED** [review]
Solid work, thanks

**Review by wise-king-sullyman — APPROVED** [review]
🔥

**Review by kmcfaul — APPROVED** [review]

**Review by srambach — APPROVED** [review]
Found a bug in core with the z-order, but this looks solid! 🗄️

**Review by tlabaj — APPROVED** [review]
LGTM

**Pull request merged**